content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -153,6 +153,34 @@ section refers to several key workload enablers and how they map to Windows.
   * `emptyDir` volumes
   * Named pipe host mounts
   * Resource limits
+  * OS field: `pod.spec.os.name` should be set to `windows` to indicate that the current pod is a Windows pod. 
+              Please note that there is no effect when setting a value for this field since it is behind a feature gate `IdentifyPodOS` and it is currently alpha.
+              {{< note >}}
```

Comment:
can we use the feature-state functionality here for consistency and also to avoid using 'currently' (which may get stale)
{{< feature-state for_k8s_version="v1.23" state="alpha" >}}

Ex: https://github.com/kubernetes/website/blob/269dad43a0d6d7df31bfc9d7b08faeaa866a4f78/content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md?plain=1#L608-L610

Reply:
@marosset - I added it [here](https://github.com/kubernetes/website/pull/30436/files#diff-29fa69c79ca964d4cc4a971d7eadbcfdfe203f97bff1a25057771c7444b147c9R157). PTAL

Reply:
If you set this field and use an older `kubectl`, I think client-side validation will fail and that does count as an effect. We should make that clear if we can.

Reply:
How do we mention it usually in our docs?